documentation/navitia.io/source/integration.rst

```diff
@@ -745,9 +745,31 @@ Parameters
 |          |                       |           | If ``arrival`` it will retrieve journeys  |                 |
 |          |                       |           | arriving before datetime.                 |                 |
 +----------+-----------------------+-----------+-------------------------------------------+-----------------+
+| nop      | traveler_type         | enum      | Define speeds and accessibility values    | standard        |
+|          |                       |           | for different kind of people              |                 |
+|          |                       |           |                                           |                 |
+|          |                       |           | * standard                                |                 |
+|          |                       |           | * slow_walker                             |                 |
+|          |                       |           | * fast_walker                             |                 |
+|          |                       |           | * luggage                                 |                 |
+|          |                       |           | * wheelchair                              |                 |
+|          |                       |           |                                           |                 |
+|          |                       |           | Each profile also automatically           |                 |
+|          |                       |           | determines appropriate first and last     |                 |
+|          |                       |           | section modes to the covered area.        |                 |
+|          |                       |           | It is possible to overload first and last |                 |
```

Comment:
Did not get that last sentence.
Is it more accurate to say that one can overload all parameters, and especially speeds, distances and modes by setting all of them specifically ?